R/download_link.R

```diff
@@ -0,0 +1,130 @@
+#' Download link
+#'
+#' @param outputId The name of the output slot that the `downloadHandler` is assigned
+#' to.
+#' @param link_text Text that will appear describing the download action
+#' (default: "Download file").
```

Comment:
Default looks to be "Download data" not "Download file"?

Reply:
I've removed the default now anyway, based on a later comments